infrastructure/terraform/variables.tf

```diff
@@ -0,0 +1,41 @@
+variable "name" {
+  description = "the name of your stack, e.g. \"demo\""
+  type        = string
+  default     = "gesund-zusammen"
+}
+
+variable "aws-region" {
+  description = "the AWS region in which resources are created"
+  type        = string
+  default     = "eu-central-1"
+}
+
+variable "aws-access-key" {
+  type = string
+}
+
+variable "aws-secret-key" {
+  type = string
+}
+
+variable "bucket_name" {
+  description = "name of the S3 bucket where the static website will be hosted, e.g. \"demo.com\""
+  type        = string
+  default     = "gesundzusammen-staging.finleap.tech"
```

Comment:
should we paste this via a secret variable?

Reply:
ok I removed the reference to staging URL and bucket